### ISSUE TITLE
Different cytosol outputs using GUI or jupyter notebook

### ISSUE THREAD

**Issue by JKesby** [MouseLand/cellpose#200]
Hi,

thanks for the program. I have been able to get my Jupyter notebook running the nuclei segmentation perfectly but am having issues with the cytosol.

Using the GUI on a sample image, I get about 7 cells with the following settings: 
cytosol = green, nuclei = red, size = 100, cell prob = -6, flow error threshold= 0.1 [as reported by the anaconda prompt].
The prompt states "7 cells found n/ plane 0 outlines processed" 
I can save the outlines and load them in imagej fine. 

Using the jupyter notebook with the following code I get 0 outlines: 
model = models.Cellpose(gpu=True, model_type='cyto')
channels = [2,1]
masks, flows, styles, diams = model.eval(img, diameter=100, channels=channels, flow_threshold=0.1,                                             cellprob_threshold=-6)

I get some outlines for other images and if I change the settings in the notebook I can get outlines for the image but it appears to be ignoring the nuclei channel (i.e., cytosol outlines do not include a nuclei). I am sure it is something simple but I am a novice, so any help would be appreciated.

As I mentioned, this is only for the cytosol functionality, I get great equivalent segmentation between the GUI and jupyter notebook when running a nuclei equivalent [using channels = [0,0].  

Cheers,
James

**Comment by carsen-stringer** [comment]
hmm I'm not sure, just to be clear channels=[2,1] means [green = channel to segment, red = nucleus channel]

and channels=[0,0] means [gray = channel to segment, no nucleus channel]

is your image green cells with red nuclei?

**Comment by JKesby** [comment]
Yeah, green cells with red nuclei. 
I have tried altering the colour and hence the code too. 

Cheers,
James

**Comment by carsen-stringer** [comment]
hey James, were you able to get this to work? I don't see any reason why these two run types would be different? Is the channel order different in the GUI vs in the notebook somehow?

for channels=[2,1], if you plt.imshow(img[:,:,1]) you should see cells and plt.imshow(img[:,:,0]) you should see nuclei

you could also try updating your cellpose to the latest version to see if there was a bug that was fixed, I'm going to close this for now but please let us know if you're still having issues

**Comment by JKesby** [comment]
Hi Carsen, 

never resolved it - I switched to running it from the command line instead.

Cheers,
James
